libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/constant/SqlType.java

```diff
@@ -38,6 +38,9 @@ public enum SqlType {
     ALTER,
     TRUNCATE,
     CREATE,
+    CALL,
+    COMMENT,
+    SET_SESSION,
```

Comment:
Is `SET_SESSION` a subset of `SET` statement? If yes, we may not add a single SET_SESSION SQL Type, it's kind of wired.

Reply:
ok

Reply:
set session should be separated from set. set session is less dangerous than set globle